bitcoin-review-podcast/cashu-and-fedimint:-ecash-bitcoin's-layer-3-and-scalability.md

```diff
@@ -0,0 +1,2499 @@
+I know people would just choose whatever they want, but.
+
+Eric: 02:04:07
+
+I think there shouldn't be actually.
+I think maybe FMPTC, that's what we use in some documentation, but in my opinion, E-cash should never be used directly.
+Like I think it's an anti-pattern to actually send e-cash in most cases.
+Maybe it makes sense for such push payments, but the problem is you lose your compatibility with the rest of the Bitcoin ecosystem.
+Ideally, what I imagine for Fedimint is that all interactions will be violating invoices.
+That way, it doesn't matter if the other side is on a Fedimint, on Cashu or on something completely different.
+And like we avoid this monoculture essentially.
+Like otherwise what happens if like one of these projects becomes too big and we now have our custom way of transferring money.
+
+NVK: 02:04:56
+
+Yeah, no, no, but like, you know, I'm just referring to sort of like the general term, right?
+Because unfortunately we can't use the word crypto anymore for cryptography.
+That's gone.
+We can't use the word token anymore because, you know, it's going to just feel like it's a shit coin, right?
+So I feel like when people who are not as technical are sort of like discussing this and discussing ideas and maybe what they want to invent as companies that add value add and whatever.
+They often need a term, right, to describe it so that it really represents, you know, this e-cash token.
+Yeah.
+
+Eric: 02:05:31
+
+Like we call it e-cash notes generally.
+
+Calle: 02:05:34
+
+Yeah, maybe notes like a bank note because it really also gives you the impression of a piece of money that you can hand around.
+
+Eric: 02:05:42
+
+We've also switched over the years multiple times.
+Like We started with coins and we went to tokens and it was apparently too shitcoiny and now we are at notes.
+
+NVK: 02:05:50
+
+So yeah, because coin implies blockchain, you know, unfortunately.
+Yeah.
+
+Rijndael: 02:05:54
+
+And for Cashu, you can say nuts.
+
+NVK: 02:05:56
+
+Yeah, I know.
+
+Calle: 02:05:57
+
+Yeah, we also say nuts sometimes.
+
+NVK: 02:05:59
+
+See, like The job that he did with all the semantics around Cashu, I think really made e-cash like happen in the minds of people in the last year.
+People started to understand it because you created all the little metaphors and all the little terms that people needed to describe the ideas.
+Right.
+So that's why I was asking.
+So like notes is a great one.
+I'm going to start using that.
+
+## ecash integration and intercompatability
+
+
+Calle: 02:06:26
+
+But can I come back to Eric, what you said about using eCash directly instead of Lightning invoices?
+I have to think more about this.
+I think I agree if you're talking about user-facing Lightning wallets, right?
+So in the best case, a cashu wallet, also a FediMint wallet, right?
```

Comment:
While the use of ', right?' at the end of a sentence makes sense when a speaker is genuinely seeking confirmation, in many instances, such as this example, it appears as a filler. In this transcript, there are 76 occurrences of ', right?', and the majority of them can be safely removed for a more streamlined transcr